src/org.learningequality.Kolibri.SearchProvider.service

```diff
@@ -0,0 +1,3 @@
+[D-BUS Service]
+Name=org.learningequality.Kolibri.SearchProvider
+Exec=/app/bin/kolibri_search_service.sh
```

Comment:
If this is going upstream this needs to be a generated file with the correct path.